src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let totalCost;
+  const pricePerDay = 40;
+
+  switch (true) {
+    case days < 3:
+      totalCost = pricePerDay * days;
+      break;
```

Comment:
you can return your value at once here
```suggestion
    case days < 3:
      return pricePerDay * days;
```